src/app/workflow/version-modal/version-modal.component.html

```diff
@@ -206,7 +206,10 @@ <h4 mat-dialog-title>{{ isPublic || !canWrite ? 'View' : 'Edit' }} {{ entryTypeT
     </fieldset>
   </div>
   <div mat-dialog-actions fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="10px">
-    <button type="button" mat-flat-button class="secondary-1" mat-dialog-close>Cancel</button>
+    <button type="button" mat-flat-button class="secondary-1" mat-dialog-close>
+      <span *ngIf="!isPublic && canWrite; else noSave">Cancel</span>
+      <ng-template #noSave>OK</ng-template>
```

Comment:
The ticket also mentions that the OK button should be a primary button. I think this means it should have the same style as the Save Changes button. 

For this PR, it has the same style as the Cancel button:
![image](https://user-images.githubusercontent.com/25287123/167422855-0c924c88-ee07-4ce5-9eb6-0a65e2906e45.png)

Save Changes button:
![image](https://user-images.githubusercontent.com/25287123/167423004-878902a6-ce8d-4a5f-ad1a-2f5e18a4d4b3.png)